esrally/track/loader.py

```diff
@@ -343,7 +344,8 @@ def prepare_track(t, cfg):
     test_mode = cfg.opts("track", "test.mode.enabled")
     for corpus in used_corpora(t, cfg):
         data_root = data_dir(cfg, t.name, corpus.name)
-        logger.info("Resolved data root directory for document corpus [%s] in track [%s] to %s.", corpus.name, t.name, data_root)
+        logger.info("Resolved data root directory for document corpus [%s] in track [%s] to %s.", corpus.name, t.name,
```

Comment:
This change looks unrelated to the PR (also, it makes the statement harder to read now). Can we revert it?